test/response_cache_handler_test.rb

```diff
@@ -15,11 +15,18 @@ def controller
   end
 
   def handler
-    @handler ||= Cacheable::ResponseCacheHandler.new(controller) do |h|
-      h.key_data       = controller.cache_key_data
-      h.version_data   = controller.cache_version_data
-      h.block          = proc { response.body = 'some text' }
-      h.cache_store    = @cache_store
+    mock_controller = controller
+    @handler ||= Cacheable::ResponseCacheHandler.new(mock_controller) do |h|
+      h.key_data              = mock_controller.cache_key_data
+      h.version_data          = mock_controller.cache_version_data
+      h.block                 = proc { response.body = 'some text' }
+      h.cache_store           = @cache_store
+      h.env                   = mock_controller.request.env
+      h.force_refill_cache    = mock_controller.force_refill_cache?
+      h.serve_unversioned     = mock_controller.serve_unversioned_cacheable_entry?
+      h.cache_age_tolerance   = mock_controller.cache_age_tolerance_in_seconds
+      h.cache_hit_server      = ->(status, _, body) { mock_controller.render status: status, plain: body }
+      h.cache_hit_brower      = ->(_, _, _) { mock_controller.head :not_modified }
```

Comment:
What is `brower`? Should this be `browser`?

Reply:
or `bowser` 🤔  ?
![image](https://user-images.githubusercontent.com/6456191/58475455-ff5e9700-811b-11e9-980f-2211525e1d37.png)